### PR TITLE
test(preferences): add unit tests for getOrSetLong logic

### PR DESCRIPTION
**Pull request by spoisseroux**
## Purpose / Description
Adds missing test coverage for `PreferenceExtensions`:
- Verifies existing Long preference retrieval
- Tests new Long value persistence
- Tests edge cases (empty strings, zero/negative longs)

## How Has This Been Tested?
- All unit tests pass (`./gradlew AnkiDroid:testPlayDebugUnitTest --tests com.ichi2.preferences.PreferenceExtensionsTest`)
- 100% coverage for modified files (`./gradlew jacocoUnitTestReport`)
- Lint/formatting checks passed

## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/preferences/PreferenceExtensionsTest.kt [hunk 1]
```diff
@@ -69,16 +76,49 @@ class PreferenceExtensionsTest {
...
         private const val MISSING_KEY = "INVALID"
         private const val LAMBDA_RETURN = "LAMBDA"
+
+        // Long constants
+        private const val LONG_VALID_KEY = "LONG_VALID"
+        private const val LONG_VALID_VALUE = 42L
+        private const val LONG_MISSING_KEY = "LONG_INVALID"
+        private const val LONG_LAMBDA_VALUE = 100L
+        private const val LONG_NEGATIVE_VALUE = -500L
+        private const val NEGATIVE_TEST_KEY = "negative_test"
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/preferences/PreferenceExtensionsTest.kt:117]: These don't seem to add value now we're in Kotlin and using extension methods, I'd personally inline them

**Review by david-allison — APPROVED** [review]
LGTM. cheers!
